flutter/lib/src/default_integrations.dart

```diff
@@ -494,3 +497,66 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
         .replaceAll('\u{0000}', '');
   }
 }
+
+/// Enrich [SentryTransaction] objects with native app data for mobile vitals.
+class MobileVitalsIntegration extends Integration<SentryFlutterOptions> {
```

Comment:
Maybe we should create a new file, the default_integrations are getting bigger, wdyt?

Reply:
Is the intention to implement the slow and frozen frames in the same integration? if so, then the name would make sense, otherwise, I'd rather call e.g. AppStartIntegration

Reply:
- Will move it out.
- Yes, the intention, for now, is to also do the frames here.